Prometheus Integartion supports ChangeFeed Metrics (#13196)
pingcap/docs: tidb-cloud/monitor-prometheus-and-grafana-integration.md

| tidbcloud_db_failed_queries_total | count | type: `planner:xxx\|executor:2345\|...`<br/>cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…`<br/>component: `tidb` | The total number of execution errors |
| tidbcloud_db_connections | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…`<br/>component: `tidb` | Current number of connections in your TiDB server |
| tidbcloud_db_query_duration_seconds | histogram | sql_type: `Select\|Insert\|...`<br/>cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…`<br/>component: `tidb` | The duration histogram of statements |

> | tidbcloud_changefeed_latency | gauge | changefeed_id | The data replication lag between the upstream and the downstream of a changefeed |

Reviewer: 需要加上单位吗？比如毫秒、秒之类的。
```suggestion
| tidbcloud_changefeed_latency | gauge | changefeed_id | The data replication latency between the upstream and the downstream of a changefeed |
```

Author: 就是 s，但是面板会自动换算成 ms 之类的。

> | tidbcloud_changefeed_latency | gauge | changefeed_id | The progress latency of data replication between the upstream and the downstream. It can also be referred to as replication lag |

Reviewer 2: ```suggestion
| tidbcloud_changefeed_latency | gauge | changefeed_id | The data replication lag between the upstream and the downstream of a changefeed |
```

> | tidbcloud_changefeed_Replica_rows | gauge | changefeed_id | The number of replica rows that ChangeFeed writes to the downstream per second |

Reviewer 2: tidbcloud_changefeed_Replica_rows 中的 R 是小写吗？

Author: 是小写

> | tidbcloud_changefeed_Replica_rows | gauge | changefeed_id | The number of replica rows that ChangeFeed writes to the downstream per second |

Reviewer 2: ```suggestion
| tidbcloud_changefeed_replica_rows | gauge | changefeed_id | The number of replicated rows that a changefeed writes to the downstream per second |
```

| tidbcloud_node_storage_capacity_bytes | gauge | cluster_name: `<cluster name>`<br/>instance: `tikv-0\|tikv-1…\|tiflash-0\|tiflash-1…`<br/>component: `tikv\|tiflash` | The disk capacity bytes of TiKV/TiFlash nodes |
| tidbcloud_node_cpu_seconds_total | count | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The CPU usage of TiDB/TiKV/TiFlash nodes |
| tidbcloud_node_cpu_capacity_cores | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The CPU limit cores of TiDB/TiKV/TiFlash nodes |
| tidbcloud_node_memory_used_bytes | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The used memory bytes of TiDB/TiKV/TiFlash nodes |
| tidbcloud_node_memory_capacity_bytes | gauge | cluster_name: `<cluster name>`<br/>instance: `tidb-0\|tidb-1…\|tikv-0…\|tiflash-0…`<br/>component: `tidb\|tikv\|tiflash` | The memory capacity bytes of TiDB/TiKV/TiFlash nodes |

## FAQ

> - Why is the value of the same metric different between Grafana and the TiDB Cloud console at the same time?

Reviewer 2: ```suggestion
- Why does the same metric have different values on Grafana and the TiDB Cloud console at the same time?
```

> - Why does the value of the same metric is different between Grafana and the TiDB Cloud console at the same time?

Reviewer 2: ```suggestion
- Why is the value of the same metric different between Grafana and the TiDB Cloud console at the same time?
```

>     The aggregation calculation logic is different between Grafana and TiDB Cloud, so the displayed aggregated values might differ. You can adjust the `mini step` configuration in Grafana to get more granular metric values.

Reviewer 2: ```suggestion
    The aggregation calculation logic is different between Grafana and TiDB Cloud, so the displayed aggregated values might differ. You can adjust the `mini step` configuration in Grafana to get more fine-grained metric values.
```

> 1. Why is there a difference in the value of the same metric at the same time between Grafana and Cloud UI?
> 
> Due to the differences in aggregation calculation logic between Grafana and the Cloud UI, there may be differences in the aggregated values displayed. Users can adjust the `mini step` configuration in Grafana to obtain more granular metric values.

Reviewer 2: ```suggestion
- Why does the value of the same metric is different between Grafana and the Cloud console at the same time?

    The aggregation calculation logic is different between Grafana and TiDB Cloud, so the displayed aggregated values might differ. You can adjust the "mini step" configuration in Grafana to get more granular metric values.
```
